modules/confiantRtdProvider.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * This module provides comprehensive detection of security, quality, and privacy threats by Confiant Inc,
+ * the industry leader in real-time detecting and blocking of bad ads
+ *
+ * The {@link module:modules/realTimeData} module is required
+ * The module will inject a Confiant Inc. script into the page to monitor ad impressions
+ * @module modules/cleanioRtdProvider
+ * @requires module:modules/realTimeData
+ */
+
+import { submodule } from '../src/hook.js';
+import { logError, insertElement, generateUUID } from '../src/utils.js';
+import * as events from '../src/events.js';
+import CONSTANTS from '../src/constants.json';
+
+/**
+ * Injects the Confiant Inc. configuration script into the page, based on proprtyId provided
+ * @param {string} propertyId
+ */
+function injectConfigScript(propertyId) {
+  const scriptElement = document.createElement('script');
+  scriptElement.type = 'text/javascript';
+  scriptElement.src = `https://cdn.confiant-integrations.net/${propertyId}/gpt_and_prebid/config.js`;
+
+  insertElement(scriptElement);
+}
+
+/**
+ * Set up page with Confiant integration
+ * @param {Object} config
+ */
+function setupPage(config) {
+  const propertyId = config?.params?.propertyId;
```

Comment:
we would also want to support one more key:
prebidExcludeBidders
which is a comma separated list of bidders which we should scan
which should be setup as
config.prebidExcludeBidders
e.g. "pubmatic,tryplelift"

Reply:
prebidNameSpace is one more key which does not have to implemented now
but will need to be supported in the future
correspond to config.prebidNameSpace
if it's less time let's implement as optional